app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -1,91 +1,115 @@
+input[type=password],
+textarea { /* Bootstrap reset */
+  &,
+  &:active,
+  &:invalid,
+  &:invalid:required,
+  &:focus,
+  &:active:focus,
+  &:invalid:focus,
+  &:invalid:required:focus {
+    border-color: $border-grey;
+    box-shadow: none;
+    color : $text-dark-grey;
+  }
+}
+
 /* autocomplete colors */
 input:-webkit-autofill{
   background-color: #fff !important;
   background-image: none !important;
 }
 
-
+/* Forms described here are only used on the public pages at the moment */
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
     color : $text-dark-grey;
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
   }
 
-  input {
-    &:invalid,
-    &:invalid:required,
-    &:invalid:required:focus {
-      box-shadow : none;
+  fieldset {
+    margin-bottom: 1em;
+    background-color: #fff;
+    position: relative; /* To correctly place the entypo icon */
 
-      border : none;
+    input {
       color : $text-dark-grey;
+      margin: 0px;
+      border-bottom-width: 0px;
+      border-radius: 0px;
     }
-  }
 
-  fieldset {
-    box-shadow : inset 0 1px 1px rgba(0,0,0,0.2), 0 1px 2px rgba(255,255,255,0.7);
-    margin-bottom : 1em;
-    background-color : #fff;
-    border : 1px solid $border-dark-grey;
-    border-radius: 5px;
-
-    input[type=text],
-    input[type=email],
-    input[type=password] {
-      box-shadow: 0 0 0 0;
-      border-radius: 0;
-      border : none;
-      background : transparent;
-      padding : 10px;
-      margin-bottom : 0;
+    .form-control {
+      font-size: 16px;
+      height: 40px;
+      padding: 10px;
+      padding-left: 40px;
     }
 
-  /* mainly bootstrap overrides */
-    .control-group {
-      margin : 0;
-      border-bottom : 1px solid $border-grey;
-
-      .control-label,
-      input[type=text],
-      input[type=password],
-      .field_with_errors label {
-        padding : 10px;
-        margin : 0;
+    .form-control:first-of-type {
+      &,
+      &:focus,
+      &:invalid,
+      &:invalid:focus,
+      &:invalid:required,
+      &:invalid:required:focus {
+        border-top-left-radius: 5px;
+        border-top-right-radius: 5px;
       }
+    }
 
-      .controls { margin-left : 100px; position : relative; }
-      .control-label { width : 80px; }
-
-      .controls .field_with_errors input {
-        background : {
-          image : image-url('invalid_fat@2x.png');
-          repeat : no-repeat;
-          position : 197px;
-          size: 20px 20px;
-        }
+    .form-control:last-of-type {
+      &,
+      &:focus,
+      &:invalid,
+      &:invalid:focus,
+      &:invalid:required,
+      &:invalid:required:focus {
+        border-bottom-left-radius: 5px;
+        border-bottom-right-radius: 5px;
+        border-bottom-width: 1px;
       }
     }
-  }
 
-  input[type=submit] {
-    text-align : center;
-    display : block;
-    width : 100%;
-    padding : 8px 0;
+    .entypo {
+      position: absolute;
```

Comment:
Properties should be ordered color, font-size, left, position, text-align, top, width